.github/workflows/test.yml

```diff
@@ -60,6 +60,11 @@ jobs:
         run: |
           sudo apt-get install -y expect
 
+      - name: Install tools on MacOS
+        if: startsWith(matrix.platform, 'macos')
+        run: |
+          brew install expect coreutils
```

Comment:
what about installing bash from brew as well so you don't have a bash old version compatibility issue in the future

Reply:
Thanks for the pointer, but it looks like the runner is already running a recent version `5.2.26(1)-release`. So we should be fine.